### DISCUSSION TITLE
One time pubkeys

### DISCUSSION THREAD

**Discussion by serrq** [KoalaSat/nostros#327] [Ideas]
Since the pubkey is a fixed and unique value, this leads to problems as well as advantages.

I ask the invention of a new method of pairing, one time pubkeys, so as not to expose the fixed one to malicious restrictions or policies of preventive injustice by the relays or even to persecute a person for the sole purpose of making it a goal of conquest or discriminate him for prejudice.

I ask for a system where from a private key you can generate public keys that can only be used once.

In short I issue a disposable public key, my contact imports it and cannot share it with anyone else once it has been activated for the first time.
